proto/p2p/p2p.proto

```diff
@@ -56,8 +57,7 @@ message Message {
     AppRequest app_request = 30;
     AppResponse app_response = 31;
     AppGossip app_gossip = 32;
-
-    PeerListAck peer_list_ack = 33;
+    // 33 is reserved until after durango activates.
```

Comment:
Should we put `reserved 33;` here?

Reply:
I don't think reserved works here (I tried)

Reply:
Turns out the reserved just can't be inside of the `oneof`. It works when outside the `oneof`